test/blackbox-tests/test-cases/install-with-var/run.t

```diff
@@ -24,3 +24,102 @@
     "_build/install/default/man/man-page-a.default.1" {"man-page-a.default.1"}
     "_build/install/default/man/man1/man-page-b.default.1"
     "_build/install/default/man/man3/another-man-page.3"
+
+Some variables are restricted in [dst] of [bin] section because evaluating
+them could cause a dependency cycle (also, most of them make no sense in [dst] anyway).
+
+  $ unset FOO
```

Comment:
Note that this is currently useless. The cram test machinery doesn't preserve the execution context between commands

Reply:
Ah, thanks. That makes sense. I actually noticed this variable being ignored, but I thought the reason was different.

Reply:
If you want to be sure it is set the way you want in the test, you can add an `env` stanza:

```
(env (_ (env-vars (FOO blah))))
```

Reply:
Thanks, I'll keep that in mind.